zebra-consensus/src/block/subsidies.rs

```diff
@@ -0,0 +1,265 @@
+//! Calculation of Block Subsidy, Funding Streams, and Founders’ Reward
+
+use super::*;
+
+use zebra_chain::{
+    block::Height,
+    parameters::{Network, NetworkUpgrade::*},
+};
+
+use crate::parameters::{fs, Params};
+
+#[cfg(test)]
+use color_eyre::Report;
+
+fn slow_start_shift() -> Height {
+    Height(Params::SLOW_START_INTERVAL / 2)
+}
+
+fn slow_start_rate() -> u32 {
+    Params::MAX_BLOCK_SUBSIDY / Params::SLOW_START_INTERVAL
+}
+
+fn is_bolossom_activated(height: block::Height, network: Network) -> Option<Height> {
```

Comment:
```suggestion
fn is_blossom_activated(height: block::Height, network: Network) -> Option<Height> {
```

Reply:
(if this function is retained)

Reply:
D'oh!. I ended up doing https://github.com/ZcashFoundation/zebra/pull/1051#discussion_r487267264 at https://github.com/ZcashFoundation/zebra/pull/1051/commits/f3f6ec63110fac434263c0c9c728a54b27b8deb0. Thanks.